src/views/dashboard.ejs

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+
+    <title>Dashboard</title>
+  </head>
+  <body>
+    <div class="container mt-4" style="text-align: center">
+      <h1>Registration Dashboard</h1>
+    </div>
+    <div
+      class="card"
+      style="
+        width: 18rem;
+        margin-left: 750px;
+        margin-top: 100px;
+        text-align: center;
+      "
+    >
+      <!-- <img src="..." class="card-img-top" alt="..."> -->
+      <div class="card-body">
+        <h5 class="card-title">Stand Registration</h5>
+        <p class="card-text">Click here for stand registration</p>
+        <a href="/register/stand" class="btn btn-primary">register stand</a>
+      </div>
+    </div>
+
+    <div
```

Comment:
Have a separate css file for styling and add media queries for responsive design.